@here/harp-mapview/lib/MapView.ts

```diff
@@ -1293,7 +1309,8 @@ export class MapView extends THREE.EventDispatcher {
 
         // Clear color.
         this.m_theme.clearColor = theme.clearColor;
-        this.renderer.setClearColor(new THREE.Color(theme.clearColor));
+        this.m_theme.clearAlpha = theme.clearAlpha;
+        this.renderer.setClearColor(new THREE.Color(theme.clearColor), theme.clearAlpha);
```

Comment:
For techniques we implemented RGBA colors. Might also make sense to combine clear color and alpha.

Reply:
as three.js handles them separately, and the clear color is accessed more frequently in the codebase, where no alpha is needed, I would leave it like that for now

Reply:
Yeah, fine for now.
It will be a bigger change in the future to treat all theme properties like technique properties, e.g. also allow dynamic expressions like clear color depending on zoomlevel.
Not sure when and if this is planned by @robertoraggi already.